README.md

```diff
@@ -1,5 +1,8 @@
 # project-sandbox
-a playground for experimentation
+* Project Sandbox will explore mapping the uses of publicly-owned vacant properties owned by the City of Los Angeles.
+* Project Sandbox is an example of using github resources to develop, document, track and manage and deploy civic projects with involved participation and collaboration from the general public from the beginning.
```

Comment:
How about:

Project Sandbox is an example of using [GitHub](http://github.com) as an open tool to share code and data with the public, manage tasks and deploy civic projects.